lib/Common/PlatformAgnostic/Arrays.h

```diff
@@ -0,0 +1,35 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+#pragma once
+
+namespace PlatformAgnostic
+{
+namespace Arrays
+{
+    class Utility
+    {
+#ifndef ENABLE_GLOBALIZATION
+        class ArraysLocale
+        {
+            WCHAR commaSeparator;
```

Comment:
We generally have been moving towards preferring `char16` over `WCHAR`, even though they're functionally identical (the windows-style ALLCAPS types look a bit out of place on Linux and macOS)